tests/publiccloud/prepare_tools.pm

```diff
@@ -102,11 +107,16 @@ sub run {
     # Create some directories, ipa will need them
     assert_script_run("img-proof list");
     record_info('img-proof', script_output('img-proof --version'));
+    my $terraform_version = '0.14.1';
+
+    # Terraform in a container
+    my $terraform_wrapper = <<EOT;
+#!/bin/sh
+podman run -v /root/:/root/ --env-host=true -w=\$PWD hashicorp/terraform:$terraform_version \$@
```

Comment:
I'd add a `--rm` to `podman run`, so that the container gets removed after execution. Also: don't we want to add `-i` as well for console interactions?

```suggestion
podman run --rm -i -v /root/:/root/ --env-host=true -w=\$PWD hashicorp/terraform:$terraform_version \$@
```

Reply:
what we will win on removing container after every call ? this will slow down test execution by time needed to re-create it again and again . On other hand we doing everything in `/root` which is mounted from host anyway so after every call container suppose to stay clean as it was before . 
Regarding `-i` I don't have strong opinion just that it was in initial version and than @jlausuch  asked me to remove it so if I will add it we can get into endless loop :)

Reply:
If you do a `podman run` then anyways a new container will be created but it will not be removed. See the following example:

    phoenix@racetrack-7290:~> podman run hashicorp/terraform version
    Resolved "hashicorp/terraform" as an alias (/home/phoenix/.cache/containers/short-name-aliases.conf)
    Trying to pull docker.io/hashicorp/terraform:latest...
    Getting image source signatures
    Copying blob c417fc24b2ac skipped: already exists  
    Copying blob 64a827a9a1f4 skipped: already exists  
    Copying blob 540db60ca938 [--------------------------------------] 0.0b / 0.0b
    Copying config 944d281445 done  
    Writing manifest to image destination
    Storing signatures
    Terraform v0.15.3
    on linux_amd64
    phoenix@racetrack-7290:~> podman run hashicorp/terraform version
    Terraform v0.15.3
    on linux_amd64
    phoenix@racetrack-7290:~> podman run hashicorp/terraform version
    Terraform v0.15.3
    on linux_amd64
    phoenix@racetrack-7290:~> podman container ls --all
    CONTAINER ID  IMAGE                                   COMMAND     CREATED         STATUS                    PORTS   NAMES
    01b4abe9090a  docker.io/hashicorp/terraform:latest    version     11 seconds ago  Exited (0) 9 seconds ago          peaceful_turing
    a279a739d120  docker.io/hashicorp/terraform:latest    version     6 seconds ago   Exited (0) 6 seconds ago          vigilant_feynman
    0990b93c1ff3  docker.io/hashicorp/terraform:latest    version     5 seconds ago   Exited (0) 4 seconds ago          sad_euler

You see three dangling containers, related to the three calls of `podman run hashicorp/terraform version`. So, omitting `--rm` does not prevent podman from creating new containers, but they will remain intact on the system.